board/piksiv3/patches/linux/0005-tty-serial-xuartps-Implement-automatic-RTC-CTS-flow-.patch

```diff
@@ -1,19 +1,9 @@
-From 2154e128f8450d285fd7b7eb8265dbe6a8c43e60 Mon Sep 17 00:00:00 2001
-From: Jacob McNamee <jacob@swiftnav.com>
-Date: Fri, 20 Jan 2017 23:03:09 -0800
-Subject: [PATCH] tty: serial: xuartps: Implement automatic RTC/CTS flow
- control
-
----
- drivers/tty/serial/xilinx_uartps.c | 25 ++++++++++++++++---------
- 1 file changed, 16 insertions(+), 9 deletions(-)
-
 diff --git a/drivers/tty/serial/xilinx_uartps.c b/drivers/tty/serial/xilinx_uartps.c
```

Comment:
Can we keep the header so we know what the patch is for?

Reply:
done